src/test/java/nl/hsac/fitnesse/fixture/slim/FileFixtureTest.java

```diff
@@ -0,0 +1,129 @@
+package nl.hsac.fitnesse.fixture.slim;
+
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class FileFixtureTest {
```

Comment:
Doesn't this unit test partially duplicate the content of 'FileFixture.wiki', which is executed as part of the acceptance test?
Do I overestimate the coverage of that test (aka no I'm wrong there is no duplication), this test adds ... as benefit, you missed that page, or ....?

Reply:
As to whether the fixture tests are more useful when they exist as a Junit file or a FitNesse wiki table, I plead neutrality. I agree it may make more sense in this project to place the tests mostly or even only in FitNesse tables.
Is that what you mean and prefer?